_includes/intro.html

```diff
@@ -32,11 +33,17 @@ <h2 class="section-headline">Contribute</h2>
       <h2 class="section-headline">Specs</h2>
       <p>
         We develop the JavaScript (formally, ECMAScript) specification
-        <a href="https://github.com/tc39/ecma262/">on GitHub</a> and at meetings every two months. We follow a
-        <a href="https://tc39.github.io/process-document/">four stage process</a> for new
-        <a href="https://github.com/tc39/proposals/">language feature proposals</a>. See our
-        <a href="https://github.com/tc39/agendas/">meeting agendas</a> and
-        <a href="https://github.com/tc39/tc39-notes/">minutes</a> to learn more.
+        <a href="https://github.com/tc39/ecma262/">on GitHub</a>. We meet every two
+        months. To learn more about the process, please take a look at the
```

Comment:
I feel like splitting off `We meet every two months.` as its own sentence makes it less clear what we're meeting about. Would `and meet to discuss proposals every two months` work?